src/App.js

```diff
@@ -1,4 +1,6 @@
 import React, { Component } from 'react';
+import { BrowserRouter, Route, Link } from 'react-router-dom';
```

Comment:
We are definitely going to have to figure out which one+ of these are unused, if any, later on in our project to clean things up!